src/api/common_api.js

```diff
@@ -105,7 +105,40 @@ module.exports = {
             }
         },
 
-        // TODO: Update to the relevant schema
+        assume_role_policy: {
+            type: 'object',
+            required: ['statement'],
+            properties: {
+                version: { type: 'string' },
+                statement: {
+                    type: 'array',
+                    items: {
+                        type: 'object',
+                        required: ['effect', 'action', 'principal'],
+                        properties: {
+                            effect: {
+                                enum: ['allow', 'deny'],
+                                type: 'string'
+                            },
+                            action: {
+                                type: 'array',
+                                items: {
+                                    type: 'string'
+                                }
+                            },
+                            principal: {
+                                // email of account
+                                type: 'array',
+                                items: {
+                                    wrapper: SensitiveString,
+                                }
```

Comment:
If this is an email then best ref to that common definition 
```suggestion
                                items: {
                                    $ref: '#/definitions/email',
                                }
```